jib-gradle-plugin/src/integration-test/java/com/google/cloud/tools/jib/gradle/JibPluginIntegrationTest.java

```diff
@@ -106,6 +127,15 @@ private static String buildToDockerDaemonAndRun(TestProject testProject, String
     return new Command("docker", "run", imageReference).run();
   }
 
+  private static String pullAndRun(String imageReference) throws IOException, InterruptedException {
```

Comment:
`pullAndRunJibBuiltImage`?  `pullAndRun` doesn't convey that it checks for a jib image layer.

Reply:
Added some javadoc as well.